### ISSUE TITLE
Cropping to 513x513 policy and inference for original size images

### ISSUE THREAD

**Issue by K-H-Ismail** [tensorflow/models#10329]
Hello, 

The documentation of Deeplab on semantic segmentation task in PascalVoc2012 (https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/pascal.md) describes how to train and validate the model on the validation set with cropped images (513x513). This doesn't describe how to make the segmentation for the test set (with the uncropped images).
The only way I found to do so is to take the original images and a batch size of 1 (in order to avoid the dimension mismatch  in tensors) and perform the inference this way. Unfortunately making so reduces the accuracy on test set (by 8 % as I tested it on the official evaluation server). Could you please describe how to do the inference on the original sized images and with a model that has been trained with a 513x513 crop option ?
